src/config/config.ts

```diff
@@ -192,3 +192,18 @@ export function initializeConfiguration(){
     }
 
 }
+
+export function fetchServerConfig(){
```

Comment:
we're using "config" a little loosely now.  The config service returns not just configuration data but user's authentication state.   Maybe we should at least split it into nodes.  @inodb what do you think?

Reply:
yeah might be nice to move the auth out, but fine for now imo